projects/plugins/jetpack/extensions/plugins/ai-assistant-plugin/hooks/use-save-to-media-library.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import { isBlobURL } from '@wordpress/blob';
+import { store as blockEditorStore } from '@wordpress/block-editor';
+import { useSelect } from '@wordpress/data';
+import { useState } from 'react';
+/**
+ * Types
+ */
+import type { BlockEditorStore } from '../../../blocks/ai-assistant/types';
+
+export default function useSaveToMediaLibrary() {
+	const [ isLoading, setIsLoading ] = useState( false );
+	const { getSettings } = useSelect( blockEditorStore, [] ) as BlockEditorStore[ 'selectors' ];
```

Comment:
Same as above, we may need to check the P2 editor to see if the const will work or if we need to put the explicit string as the store name.